### DISCUSSION TITLE
Global hotkey - Open KeepassXC

### DISCUSSION THREAD

**Discussion by petvo78** [keepassxreboot/keepassxc#7609] [Ideas]
How can I open KeepassXC with a **global hotkey** (not autotype)?

Looking everywhere, also trying with Autohotkey script but cant it to work.
I would like to have a global hotkey to open the application (like Keepass original). This is the only thing that stops me for switching to KeepassXC.

Possible? :)

**Comment by droidmonkey** [comment]
Use autohotkey to just run keepassxc again. It will pop up the already running instance.

**Reply by petvo78** [reply]
Thanks, that worked!
`run c:\mypath\KeePassXC.exe`

**Comment by ghost** [comment]
You don't even need autohotkey, you could just pin KeepassXC to your taskbar then open it with Win Key + <position in taskbar #>
